src/flux/helpers.coffee

```diff
@@ -82,6 +84,12 @@ CrudConfig = ->
       @_local[result.id] = result
       @emitChange()
 
+    delete: (id) ->
+      @_asyncStatus[id] = DELETING
+
+    deleted: (result, id) ->
+      @_asyncStatus[id] = DELETED
+      @_local[id] = null
```

Comment:
This should probably be `delete @_local[id]` and you should call `@emitChange()` so any views (like a listing of plans) gets updated. I don't think that view exists yet, but it will in this sprint.